templates/spa/README.md

```diff
@@ -22,16 +22,18 @@ npm run dev
 
 ## Production
 
-When you are ready yo build a production version of your app, `npm run build` will generate your assets and an `index.html` for the SPA.
+When you are ready to build a production version of your app, `npm run build` will generate your assets and an `index.html` for the SPA.
 
 ```shellscript
 npm run build
 ```
 
-You can serve this from any server of your choosing, for a simple example, you could use [http-server](https://www.npmjs.com/package/http-server):
+You can serve this from any server of your choosing. The server should be configured to serve multiple paths from a single root `/index.html` file (commonly called "SPA fallback"). Other steps may be required if the host doesn't directly support this functionality.
+
+For a simple example, you could use [sirv-cli](https://www.npmjs.com/package/sirv-cli):
 
 ```shellscript
-npx http-server build/client/
+npx sirv-cli build/client/ --single --ignores "^/assets/"
```

Comment:
Do we need `--ignores "^/assets/"` here?  It seemed to work ok for me without that?

Reply:
It does work without it, but the http server would then return `200 OK` along with the `index.html` contents for any missing assets.

Reply:
For example, I'm not sure how Remix detects that it needs to perform a reload because the files on the server got updated. Would that feature be impacted if the assets never `404`?

Reply:
I should have tested this. Missing asset files will 404 because they have a file extension.

This could be considered a limitation of the `--single` mode where any path with an extension not fall-back to `index.html`.